controls/V-233519.rb

```diff
@@ -49,6 +49,7 @@
 
 	describe postgres_hba_conf(pg_hba_conf_file) do
 		its('auth_method') { should_not include 'password' }
+		its('auth_method') { should_not include 'md5' }

```

Comment:
You can simplify this with 

```ruby
its('auth_method') { should_not include %r{password|md5}i }
```

Reply:
The `include` matcher supports regex

Reply:
Thank you, will do.

Reply:
Hey @aaronlippold, Eugene and I ran your recommended test and this is the output we see for a failure:
```
  ×  V-233519: If passwords are used for authentication, PostgreSQL must transmit only encrypted representations of
          passwords.
     ×  Postgres Hba Config /var/lib/pgsql/12/data/pg_hba.conf auth_method is expected not to include /password|md5/i
     expected ["peer", "ident", "ident", "peer", "ident", "md5"] not to include /password|md5/i
     Diff:
     @@ -1,2 +1,2 @@
     -[/password|md5/i]
     +["peer", "ident", "ident", "peer", "ident", "md5"]
```

This might be confusing to the user if they don't understand the syntax for `/password|md5/i`

Since it's just one more line of test, I believe using another `its` statement would make the output cleaner to read:
```
  ×  V-233519: If passwords are used for authentication, PostgreSQL must transmit only encrypted representations of
          passwords. (1 failed)
     ✔  Postgres Hba Config /var/lib/pgsql/12/data/pg_hba.conf auth_method is expected not to include "password"
     ×  Postgres Hba Config /var/lib/pgsql/12/data/pg_hba.conf auth_method is expected not to include "md5"
     expected ["peer", "ident", "ident", "peer", "ident", "md5"] not to include "md5"
```

Would you be okay with @damif5 updating the control to this:

```
describe postgres_hba_conf("#{input('pg_hba_conf_file')}") do
  its('auth_method') { should_not include 'password' }
  its('auth_method') { should_not include 'md5' }
end
```

Reply:
Sounds good

Reply:
Thank you.